external/fv3fit/fv3fit/keras/_models/dense.py

```diff
@@ -84,7 +113,8 @@ def train_dense_model(
             dataset_sequence=[validation_batches[0]],
             unstacked_dims=["z"],
             n_halo=0,
-        )[0]
+        )
```

Comment:
Does the code not break if you pass a sequence of (X, y) pairs as validation_data to `model.fit`? If it doesn't, this is definitely better, but I'm surprised.

Keep in mind that the model evaluates on the validation data every batch, so this could slow down training considerably depending on the case. Mostly something to keep in mind when you're configuring runs. Also you may want to communicate this to the team as a backwards-incompatible change (before it ignored all but the first batch).

Reply:
Interestingly, it does not.

Reply:
Actually there was another line in addition to this that ensured only the first item in the sequence was used (ln 113 above).